libs/Microsoft.MixedReality.WebRTC/Microsoft.MixedReality.WebRTC.csproj

```diff
@@ -39,6 +39,13 @@
     <NativeUnityPluginSourceFiles Include="$(OutputPath)\Microsoft.MixedReality.WebRTC.dll;$(OutputPath)\Microsoft.MixedReality.WebRTC.pdb" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Will this work even if the NuGet package is generated using a nuspec (which is the case for the official packages)?

Reply:
@fibann I must have assumed you were using the SDK style CSPROJ way of building packages. I removed the `.props` file from the `.cproj` and moved it to the `nuget` folder. I'm not sure about the relative paths I placed in the `.nuspec` though. Tried to figure it out based on your YAML pipelines but couldn't really see if the .props file should be moved anywhere.